src/Core/Services/Implementations/PolicyService.cs

```diff
@@ -53,25 +55,52 @@ public class PolicyService : IPolicyService
                 var currentPolicy = await _policyRepository.GetByIdAsync(policy.Id);
                 if (!currentPolicy?.Enabled ?? true)
                 {
-                    if (currentPolicy.Type == Enums.PolicyType.TwoFactorAuthentication)
+                    var kickableOffenses = new List<Enums.PolicyType> {
+                        Enums.PolicyType.TwoFactorAuthentication,
+                        Enums.PolicyType.OnlyOrg
+                    };
+
+                    if (kickableOffenses.Contains(currentPolicy.Type))
                     {
                         Organization organization = null;
                         var orgUsers = await _organizationUserRepository.GetManyDetailsByOrganizationAsync(
                             policy.OrganizationId);
                         foreach (var orgUser in orgUsers.Where(ou =>
                             ou.Status != Enums.OrganizationUserStatusType.Invited &&
-                            ou.Type != Enums.OrganizationUserType.Owner))
+                            ou.Type != Enums.OrganizationUserType.Owner && ou.UserId != savingUserId))
```

Comment:
Would we not kick out the user setting the policy if they are not the owner BUT are in violation? Seems silly but not sure what's appropriate here. Will need @kspearrin to weight in.

Reply:
This was actually a check I just moved up from [line 65](https://github.com/bitwarden/server/pull/962/files/de295deb245da6ebd54e1b32c7c12b58ac9adb62#diff-928f61507e60ccef15abcc2ae6b96ae6125cd05a80e6ef2f34439f7914c29b36L65), so it is something we were already filtering on for 2FA.